integration-tests/emulated/tests/bridges/bridge-hub-polkadot/src/tests/snowbridge.rs

```diff
@@ -459,6 +461,184 @@ fn send_token_from_ethereum_to_asset_hub() {
 	});
 }
 
+/// Tests sending ether from Ethereum to Asset Hub and back to Ethereum
+#[test]
+fn send_eth_asset_from_asset_hub_to_ethereum() {
+	let assethub_sovereign = BridgeHubPolkadot::sovereign_account_id_of(
+		BridgeHubPolkadot::sibling_location_of(AssetHubPolkadot::para_id()),
+	);
+
+	BridgeHubPolkadot::fund_accounts(vec![
+		(assethub_sovereign.clone(), INITIAL_FUND),
+		(RelayTreasuryPalletAccount::get(), INITIAL_FUND),
+	]);
+	AssetHubPolkadot::fund_accounts(vec![
+		(AssetHubPolkadotReceiver::get(), INITIAL_FUND),
+		(ethereum_sovereign_account(), INITIAL_FUND),
+	]);
+
+	let ether_location: Location = (Parent, Parent, EthereumNetwork::get()).into();
+
+	AssetHubPolkadot::execute_with(|| {
+		type RuntimeEvent = <AssetHubPolkadot as Chain>::RuntimeEvent;
+		type RuntimeOrigin = <AssetHubPolkadot as Chain>::RuntimeOrigin;
+
+		// Set base transfer fee to Ethereum on AH.
+		assert_ok!(<AssetHubPolkadot as Chain>::System::set_storage(
+			<AssetHubPolkadot as Chain>::RuntimeOrigin::root(),
+			vec![(BridgeHubEthereumBaseFee::key().to_vec(), AH_BASE_FEE.encode())],
+		));
+
+		// Register Ether
+		assert_ok!(<AssetHubPolkadot as AssetHubPolkadotPallet>::ForeignAssets::force_create(
+			RuntimeOrigin::root(),
+			ether_location.clone(),
+			ethereum_sovereign_account().into(),
+			true,
+			MIN_ETHER_BALANCE,
+		));
+
+		assert_expected_events!(
+			AssetHubPolkadot,
+			vec![
+				RuntimeEvent::ForeignAssets(pallet_assets::Event::ForceCreated { .. }) => {},
+			]
+		);
+	});
+
+	// Send Ether from Bridge Hub
```

Comment:
```suggestion
	// Send Ether from Bridge Hub (simulates received Command from Ethereum)
```

Reply:
Addressed in e946e286e499a6b16890f22b40738fa12aff6b7c